renovate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": [
+    "config:base",
+    ":maintainLockFilesWeekly",
+    ":automergeRequireAllStatusChecks"
+  ],
+  "assignAutomerge": true,
+  "reviewers": ["KillianHmyd", "GuillaumeRahbari"],
```

Comment:
😭

Reply:
Do you want me to put the whole team ? ^^'

Reply:
No, don't worry 😄  Most of the time it should be an automatic merge.
Btw I saw on semantic release a step to check that dependencies are compatible with a specific node version : https://github.com/semantic-release/semantic-release/runs/4143844214?check_suite_focus=true
Don't know if it would be relevant here

Reply:
I think that yarn is doing it on his own. Try to use node 12 on the repo and to do a yarn install. In my computer it fails cause of the dependencies not being compatible

Reply:
Maybe I could add it

Reply:
But should we check with every node version then ? 🤔

Reply:
I think this is related to the engines in the package json

Reply:
I don't know if it is useful honestly

Reply:
I updated our engine to use node 14. If I am using node 12, yarn fails on the install. I think it could be enough